gdcdictionary/schemas/serious_adverse_event.yaml

```diff
@@ -0,0 +1,192 @@
+#       backref: saes
+#       label: data_from
+#       target_type: sae_workflow
+#       multiplicity: many_to_one
+#     - name: sae_files
+#       backref: saes
+#       label: data_from
+#       target_type: sae_file
+#       multiplicity: one_to_many
+links:
+  - name: protocols
+    backref: sae_files
+    label: recorded_at
+    target_type: protocol
+    multiplicity: many_to_one
+    required: true
+
+required:
+  - submitter_id
+  - type
+  - protocols
+  - quarter_of_sae
+
+uniqueKeys:
+  - [id]
+  - [project_id, submitter_id]
+
+properties:
+  type:
+    type: string
+  id:
+    $ref: "_definitions.yaml#/UUID"
+    systemAlias: node_id
+  state:
+    $ref: "_definitions.yaml#/state"
+  #REDCAP ids
+  submitter_id:
+    type:
+      - string
+      - "null"
+  protocols:
+    $ref: "_definitions.yaml#/to_one"
+  project_id:
+    $ref: "_definitions.yaml#/project_id"
+  created_datetime:
+    $ref: "_definitions.yaml#/datetime"
+  updated_datetime:
+    $ref: "_definitions.yaml#/datetime"
+  # is_final_report:
+  #   description:
+  #     If this is the finalized report? (option on follow up form)
+  #   enum:
+  #     - 'Yes'
+  #     - 'No'
+  #     - Not reported
+  quarter_of_sae:
+    type: string #change to number when we figure out how missing/null vals are coded
+    description: >
+      The quarter and year in which serious adverse event was recorded XXXXqX (e.g., 2020q1 for year 2020, Quarter 1).
+    type: string
+  #propreties with suffix of "_occurred" are a part of
+  # "Check which of the following occurred"
+  # death_occurred:
+  #   enum:
+  #     - 'Yes'
+  #     - 'No'
+  #     - Not reported
+
+  # life_threatening_occurred:
+  #   enum:
+  #     - 'Yes'
+  #     - 'No'
+  #     - Not reported
+  # hospitalization_occurred:
+  #   enum:
+  #     - 'Yes'
+  #     - 'No'
+  #     - Not reported
+  # disability_occurred:
+  #   enum:
+  #     - 'Yes'
+  #     - 'No'
+  #     - Not reported
+
+  # birth_defect_occurred:
+  #   enum:
+  #     - 'Yes'
+  #     - 'No'
+  #     - Not reported
+
+  # required_treatment_occurred:
+  #   enum:
+  #     - 'Yes'
+  #     - 'No'
+  #     - Not reported
+
+  is_related_to_study:
+    description: >
+      Is this serious adverse event related to the study?
+      Note,  possible means:
+      There is a reasonable possibility that 
+      the incident, experience, or outcome may have been 
+      caused by the procedures involved in the research. 
+    enum:
+      - 'Yes'
+      - 'No'
+      - Not reported
+
+  was_expected:
+    enum:
+      - 'Yes'
+      - 'No'
+      - Not reported
+  #get hubs who are taking MOUD intervention
+  on_opioid_medication:
+    description: >
+      Was client taking a medication for opioid use disorder (MOUD)?
+      This is derived from hubs who are are using MOUD intervention.
+    enum:
+      - 'Yes'
+      - 'No'
+      - Not reported
+
+#identical fields in demographic.yaml. *Gender is slightly different than any of the fields
```

Comment:
`race` and `hispanic` definitions could be moved to `_definitions.yaml` and referenced in `demographic.yaml` and `serious_adverse_event.yaml`, instead of duplicating them and risking a mismatch in the future.
I would advise doing the same for `gender` even if the values that are effectively used in this node are only a subset of the allowed values.

Reply:
Appreciate it -- will do tomorrow and let you know said change.

Reply:
included in the _definitions.yaml. However, the build is not passing.... 

https://app.travis-ci.com/github/uc-cdis/JCOIN_datadictionary/builds/238756571 says:
ERROR: Failure: RefResolutionError (Unresolvable JSON pointer: 'race')

Yet, all 23 of my local dictionaryutils tests pass...does the local test not test for references?

Reply:
@paulineribeyre  Looks like the checks passed! So, resolving convo and I think its ready for a merge. Thanks for your help